ops/log.py

```diff
@@ -26,7 +26,7 @@
 
 
 class JujuLogHandler(logging.Handler):
-    """A handler for sending logs to Juju via juju-log."""
+    """A handler for sending logs to Juju via Juju-log."""
```

Comment:
Why the change here? The hook command is called "juju-log".

Reply:
ah, my bad. Was overly enthusiastic in batch-replacing after I was reminded that the official name is capitalized. Will roll that one back.